products/views.py

```diff
@@ -1,3 +1,71 @@
+from django.contrib.admin.views.decorators import staff_member_required
 from django.shortcuts import render
+from django.core.exceptions import ObjectDoesNotExist
+from django.shortcuts import render, get_object_or_404
+from django.views.generic import View
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
+from django.http import HttpResponseRedirect
 
+from models import Book, CartItems, Cart, UserAddress
+from datetime import datetime, timezone
 # Create your views here.
+@login_required
+def add_to_cart(request, book, user):
+    carts = Cart.objects.get_object_or_404(user=user)
+    if carts.exists():
+        cart = carts[0]
+        cartItems = CartItems.objects.filter(cart=cart, book=book).first()
+        if cartItems.exists():
+            cartItems.quantity += 1
+            cartItems.save()
+            messages.info(request, 'Book quantity updated')
+        else:
+            cartItems.objects.create(cart=cart,book=book, quantity=1)
+            messages.info(request, 'Succesfully added book')
+    else:
+        currTime = timezone.now()
```

Comment:
`datetime.timezone` doesn't have member `now()`. Perhaps you meant `datetime.datetime.now()`?